config/routes.rb

```diff
@@ -3,4 +3,6 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customer_items#create"
```

Comment:
A more RESTful path here would be POST to  "/customers/:id/items" to indicate that you are creating the relationship between a customer and an item. It's helpful to include both "customer" and "item" in the path to indicate the relationship rather than just "customers"